diff --git a/tests/featuretests/profiler/utils/csv_parser.h b/tests/featuretests/profiler/utils/csv_parser.h
index 65ee441..d3e3397 100644
--- a/tests/featuretests/profiler/utils/csv_parser.h
+++ b/tests/featuretests/profiler/utils/csv_parser.h
@@ -31,6 +31,7 @@ THE SOFTWARE.
 #include <map>
 #include <string>
 #include <vector>
+#include <cstdint>
 
 /* --------------------------------------------------------------------------*/
 /**
