diff --git a/cmake/virtualenv.cmake b/cmake/virtualenv.cmake
index 951755ae..10dad244 100644
--- a/cmake/virtualenv.cmake
+++ b/cmake/virtualenv.cmake
@@ -1,7 +1,7 @@
 # find_package(PythonInterp)
 # # TODO: Check PYTHON_VERSION_MAJOR
 
-find_program(VIRTUALENV_PYTHON_EXE ${python})
+find_program(VIRTUALENV_PYTHON_EXE venv ${python})
 if(NOT VIRTUALENV_PYTHON_EXE)
     # look for non default name
     if(${python} MATCHES "python3")
